Makefile

```diff
@@ -12,11 +12,11 @@ test-go:
 	
 .PHONY: run-go
 run-go:
-	SERVE_FROM_FS=true cd desktopcollector; go run ./...
+	cd desktopcollector; go run ./... --dev
```

Comment:
Can you double-check that this is working? My read of the code is that a `true` value doesn't get passed to the `Server`